build.psm1

```diff
@@ -1602,14 +1602,14 @@ function Show-PSPesterError
         $description = $testFailure.description
         $name = $testFailure.name
         $message = $testFailure.failure.message
-        $StackTrace = $testFailure.failure."stack-trace"
+        $stack_trace = $testFailure.failure."stack-trace"
```

Comment:
This is to fix a warning from PSSA -- `$StackTrace` is a built-in variable, so should use a different name.